src/SharedElements.tsx

```diff
@@ -0,0 +1,237 @@
+                if (!expandEventSent) {
+                    expandHandler();
+                    setExpandEventFired(true);
+                }
+                setHasBeenExpanded(!hasBeenExpanded);
+            }}
+        >
+            <span
+                className={css`
+                    color: ${news[400]};
+                    display: block;
+                    ${body.medium({
+                        lineHeight: 'tight',
+                        fontWeight: 'bold',
+                    })};
+                `}
+            >
+                {sectionTitle}
+            </span>
+            <h4
+                className={css`
+                    ${headline.xxxsmall({
+                        fontWeight: 'medium',
+                    })};
+                    margin: 0;
+                    line-height: 22px;
+                `}
+            >
+                {title}
+            </h4>
+            <span className={showHideStyling}>
+                {!hasBeenExpanded && (
+                    <span className={iconSpacing}>
+                        <span className={plusStyling}>
+                            <SvgPlus />
+                        </span>
+                        Show
+                    </span>
+                )}
+                {hasBeenExpanded && (
+                    <span className={iconSpacing}>
+                        <span className={minusStyling}>
+                            <SvgMinus />
+                        </span>
+                        Hide
+                    </span>
+                )}
+            </span>
+        </summary>
+    );
+};
+
+///
+
+/// LIKE/DISLIKE FEEDBACK FOOTER
+const footerStyling = css`
+    font-size: 13px;
+    line-height: 16px;
+    display: flex;
+    justify-content: flex-end;
+`;
+
+const buttonStyling = css`
+    display: inline-flex;
+    cursor: pointer;
+    align-items: center;
+    justify-content: center;
+    background: black;
+    color: white;
+    border-style: hidden;
+    border-radius: 100%;
+    margin: 0 0 0 5px;
+    padding: 0;
+    width: 28px;
+    height: 28px;
+    :hover {
+        background: ${news[400]};
+        border-color: ${news[400]};
+    }
+
+    :focus {
+        border: none;
+    }
+`;
+
+// Currently no thumb icon in src-icons so a path is needed
+const ThumbImage = () => {
+    return (
+        <svg
+            className={css`
+                width: 16px;
+                height: 16px;
+            `}
+            viewBox="0 0 40 40"
+        >
+            <path
+                fill="#FFF"
+                d="M33.78 22.437l-4.228 13.98L27.93 37.5 5.062 34.14V15.503l7.8-1.517L24.354 2.5h1.624L28.9 5.426l-4.548 8.67h.107l10.477 1.31"
+            ></path>
+        </svg>
+    );
+};
+
+export const Footer = ({
+    likeHandler,
+    dislikeHandler,
+}: {
+    likeHandler: () => void;
+    dislikeHandler: () => void;
+}): JSX.Element => {
+    const [showFeedback, setShowFeedback] = useState(false);
```

Comment:
Maybe rename this `showThankYou` or flip it round so that it actually is `showFeedback` (if by "feedback" we mean the thumbs up/down buttons).